repos/spack_repo/builtin/packages/simsipm/package.py

```diff
@@ -43,6 +43,8 @@ class Simsipm(CMakePackage):
     depends_on("python@3.6:", when="+python", type=("build", "run"))
     depends_on("py-pybind11", when="+python", type=("build", "link"))
 
+    conflicts("arch=aarch64", msg="package uses x86 internals and does not support aarch64")
+
```

Comment:
Just ran into this as well. Would you be interested in a patch instead? This appears fixed in main, https://github.com/EdoPro98/SimSiPM/commit/b08cc27151d70298b43ca7e216add9e46c056b04

```suggestion
    conflicts("arch=aarch64", msg="SimSiPM < 2.1.0 uses x86 internals and does not support aarch64", when="@:2.0")
    patch(
        "https://github.com/EdoPro98/SimSiPM/commit/b08cc27151d70298b43ca7e216add9e46c056b04.patch?full_index=1",
        sha256="354208cd110976a948e4770d7ae086cff367cef74534d07c22524f107ffbb7fd",
        when="@2.1.0",
    )

```

Reply:
Nice, I guess, I opened this just the week too early :D 

Did you check whether this builds with the patch? Otherwise, I could try do that.

Reply:
> Nice, I guess, I opened this just the week too early :D

Or the developer is keeping an eye on spack or key4hep issue trackers :-)

> Did you check whether this builds with the patch? Otherwise, I could try do that.

I haven't tested the build. I don't have an arm system easily accessible locally, so I'm having to get this into github workflows to test: https://github.com/eic/containers/actions/runs/19217416962/job/54943884917?pr=21 (without the patch).

Reply:
Rather than figuring out how to cherry-pick from your fork as a new remote, I just changed the version to the commit of the patch: https://github.com/eic/containers/pull/21/commits/4c7acf4d18e72c76f6985c5cb388e29add5151b0. If that builds, it'll be a good sign that this patch indeed works. Just keep an eye on the pipeline at https://github.com/eic/containers/actions/runs/19236407689/job/54988113769?pr=21. If that passes, I'll mark as approved here too.

Reply:
Thanks for checking.